contracts/ERC721MOperatorFilterer.sol

```diff
@@ -50,4 +54,22 @@ contract ERC721MOperatorFilterer is ERC721M, DefaultOperatorFilterer {
     ) public payable override(ERC721A, IERC721A) onlyAllowedOperator(from) {
         super.safeTransferFrom(from, to, tokenId, data);
     }
+
+    function setTradable(bool tradable) external onlyOwner
+    {
+        _tradable = tradable;
+    }
+
+    function _beforeTokenTransfer(
+        address from,
+        address to,
+        uint256 tokenId
+    ) internal virtual override(ERC721) {
+        if (
+            _tradable != true
+        ) {
+            revert("Transfers are currently disabled");
```

Comment:
prefer to revert an error: https://github.com/magicoss/erc721m/blob/main/contracts/IERC721M.sol#L7-L27